my.css

```diff
@@ -0,0 +1,165 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header, .collection, {
+	z-index: 2;
+	position: relative;
+	color: white;
+}
+.header {
+	position: absolute;
+	top: 0;
+	left: 0;
+	right: 0;
+	display: flex;
+	justify-content: space-between;
+}
+.wraper::before {
+	content: '';
+	position: absolute;
+	left: 0;
+	top: 0;
+	width: 400px;
+	height: 900px;
+	background-color: black;
+	z-index: -1;
+}
+.burgermenu {
+  position: relative; 
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center; 
+  width: 92px;
+  height: 92px;
+  background: lightgray;
+}
+.berger__line {
+	margin: 4px;
+	width: 34.7px;
+    height: 3.48px;
+	background-color: #000;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)